oauthenticator/azuread.py

```diff
@@ -38,6 +39,12 @@ def _tenant_id_default(self):
     def _username_claim_default(self):
         return 'name'
 
+    user_groups_claim = Unicode(config=True)
```

Comment:
I'd like there to be a help string for this describing the configuration better in our documentation.

Reply:
Done.